Formula/influxdb.rb

```diff
@@ -44,8 +44,8 @@ class Influxdb < Formula
   # NOTE: The version/URL here is specified in scripts/fetch-ui-assets.sh in influxdb.
   # If you're upgrading to a newer influxdb version, check to see if this needs upgraded too.
```

Comment:
Can we read `scripts/fetch-ui-assets.sh` to make sure we have the right version of `ui-assets`?